ArduPlane/Log.cpp

```diff
@@ -252,6 +252,30 @@ const struct LogStructure Plane::log_structure[] = {
     LOG_COMMON_STRUCTURES,
     { LOG_STARTUP_MSG, sizeof(log_Startup),         
       "STRT", "QBH",         "TimeUS,SType,CTot", "s--", "F--" },
+
+// @LoggerMessage: CTUN
+// @Description: Control Tuning information
+// @Field: TimeUS: microseconds since system startup
+// @Field: NavRoll: desired roll
+// @Field: Roll: achieved roll
+// @Field: NavPitch: desired pitch
+// @Field: Pitch: achieved pitch
+// @Field: ThrOut: scaled output throttle
+// @Field: RdrOut: scaled output rudder
+// @Field: ThrDem: demanded speed-height-controller throttle
+// @Field: Aspd: airspeed estimate
+
+// @LoggerMessage: PIQR,PIQP,PIQY,PIQA
```

Comment:
That I can't do given the way I've currently got these implemented.

For these in particular I'm hoping to replace all of the separate messages with instance numbers - so you'd graph `PID["roll"].p` rather than `PIDR.p`.

I can go back and try harder with the markup - but it might end up just being copy/paste :-(

Reply:
no, how about just in the HEADING that shows in the wiki somehow

Reply:
After merge, I can add to the @Description field....